### DISCUSSION TITLE
Images Not Downloading

### DISCUSSION THREAD

**Discussion by pencapo** [izqalan/cy-client#55] [Q&A] [answered]
Hi, I'm very new to Github so apologies if I'm doing/not doing something really dumb and basic. I've got the downloader working, but when I add a URL to the list it just creates a folder but doesn't actually download any images. The folders it creates have the correct title from the gallery.

It doesn't throw up any error messages. It seems to all be working correctly, so I think I've gone off track somewhere. Can anyone tell me where I'm going wrong?

P.S. There are definitely images in the cyberdrop galleries.

Thanks.

**Comment by izqalan** [comment, marked as answer]
Down speed to cyberdrop has been slow in these past few days. Nothing we can do about it, sorry.

**Reply by pencapo** [reply]
Ah okay, all good then I guess. Thanks!

**Comment by izqalan** [comment]
@pencapo . @Anequit has made some improvement regarding download speed. Could you try it out and let me know how to goes?

Download: https://github.com/izqalan/cy-client/releases/tag/2.2

**Reply by Anequit** [reply]
Yeah I reworked how files are downloaded in general, so slow files still download. I've ran the downloader overnight with like 15 albums and it still going no problem. 6 or so of these albums are EXTREMELY slow for some reason, but it didn't fail on any of them.

**Reply by Anequit** [reply]
so far I've downloaded 14 gb worth of files with none of them failing.

**Comment by OffclAlanSmithee** [comment]
After 5 failed files it stops downloading
